### PR TITLE
Enhancement: Update friendsofphp/php-cs-fixer

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] removes `fabpot/php-cs-fixer` and requires `friendsofphp/php-cs-fixer`
* [x] updates `.travis.yml` and `.php_cs`
* [x] runs `php-cs-fixer`

💁‍♂️ For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/compare/v1.11.2...v2.0.0.

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -1,18 +1,25 @@
 <?php
 
-$finder = Symfony\CS\Finder\DefaultFinder::create()
+$finder = PhpCsFixer\Finder::create()
```
Comment (localheinz) [inline .php_cs]: Namespace has been renamed and collapsed.

File: .php_cs [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
-    ->level(Symfony\CS\FixerInterface::PSR2_LEVEL)
-    ->fixers([
-        '-psr0',
-        'multiline_array_trailing_comma',
-        'ordered_use',
-        'short_array_syntax',
-        'unused_use',
+    ->setRules([
+        '@PSR2' => true,
+        'array_syntax' => [
```
Comment (localheinz) [inline .php_cs]: Fixers have become rules, some of which can be configured, others can just be enabled or disabled,

File: .php_cs [hunk 3]
```diff
@@ -1,18 +1,25 @@
...
+            'syntax' => 'short'
+        ],
+        'no_unused_imports' => true,
+        'ordered_imports' => true,
+        'psr0' => false,
+        'trailing_comma_in_multiline_array' => true,
```
Comment (localheinz) [inline .php_cs]: Keeping rules sorted alphabetically makes it easier to maintain rules.

File: .travis.yml [hunk 4]
```diff
@@ -38,7 +40,7 @@ before_script:
 
 script:
   - if [[ "$COLLECT_COVERAGE" == "true" ]]; then vendor/bin/phpunit --coverage-clover build/logs/clover.xml; else vendor/bin/phpunit; fi
-  - if [[ "$WITH_CS" == "true" ]]; then vendor/bin/php-cs-fixer fix --verbose --diff; fi
+  - if [[ "$WITH_CS" == "true" ]]; then vendor/bin/php-cs-fixer fix --verbose --diff --dry-run; fi
```
Comment (localheinz) [inline .travis.yml]: Need to use the `--dry-run` option to fail the build if fixes need to be applied.

**Comment by chartjes** [comment]
Thanks @localheinz

**Pull request merged**